### PR TITLE
Feautre/persistance

### PR DESCRIPTION
**Pull request by isambard217**

### CODE REVIEW

**Comment by isambard217** [comment]
[Spring database Persistance](https://trello.com/c/HmOPuZQ2/2-spring-database-persistance)

File: src/test/java/com/ops/base/RestBluePrint/RestBluePrintApplicationTests.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 	@Test
 	public void contextLoads() {
+
```
Comment (AlhaythamElhassan) [inline src/test/java/com/ops/base/RestBluePrint/RestBluePrintApplicationTests.java]: why this lines

File: bin/main/application.properties [hunk 2]
```diff
@@ -1 +1,8 @@
 server.port=8080
+
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url = jdbc:mysql://localhost:3306/test
+spring.datasource.username=admin2
+spring.datasource.password=password
+
```
Comment (AlhaythamElhassan) [inline bin/main/application.properties]: At the end of each file we need only one new line
https://robots.thoughtbot.com/no-newline-at-end-of-file

File: bin/main/application.properties [hunk 3]
```diff
@@ -1 +1,9 @@
 server.port=8080
+
+spring.jpa.hibernate.ddl-auto= update
```
Comment (AlhaythamElhassan) [inline bin/main/application.properties]: why hibernate?

**Pull request merged**